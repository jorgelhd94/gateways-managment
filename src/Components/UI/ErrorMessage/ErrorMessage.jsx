import React from 'react'

const ErrorMessage = props => {
  return (
    <div className='block color-red-500'>
        {props.children}
    </div>
  )
}

export default ErrorMessage