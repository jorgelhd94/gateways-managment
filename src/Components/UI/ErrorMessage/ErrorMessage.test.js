import React from 'react';
import renderer from 'react-test-renderer';
import ErrorMessage from './ErrorMessage';

test('Error Message render correctly', () => {
  const component = renderer.create(<ErrorMessage>Error</ErrorMessage>);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
