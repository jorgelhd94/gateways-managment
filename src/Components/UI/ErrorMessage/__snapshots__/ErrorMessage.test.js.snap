// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error Message render correctly 1`] = `
<div
  className="text-left text-red-500"
>
  Error
</div>
`;
